app/models/agent_bot.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: agent_bots
+#
+#  id           :bigint           not null, primary key
+#  description  :string
+#  name         :string
+#  outgoing_url :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  account_id   :integer
+#  user_id      :integer
+#
+
+class AgentBot < ApplicationRecord
+  validates :account_id, presence: true
```

Comment:
I don't think this needs to be scoped under account, AgentBot is more like a config and it should not be specific to an account, whatever custom configuration is required should be done at the inbox level

Reply:
User and account is removed.